### PR TITLE
feat(agora): migrate GCT (AG-1602)

### PR DESCRIPTION
**Pull request by sagely1**
## Description
Migration of GCT and gene details page
Also migrates charts (except box plot)
miscellaneous fixes (including api)

## Existing issues
- conversion of components to standalone components and lazy-loading has resulted in a bug in the loading overlay component (to be fixed in a future PR)
- box plot chart utilized in monorepo will be incorporated in a future PR

### CODE REVIEW

File: libs/agora/charts/src/lib/median-barchart/median-barchart.component.ts [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  ViewEncapsulation,
+} from '@angular/core';
+import * as d3 from 'd3';
+
+import { HelperService } from '@sagebionetworks/agora/services';
+import { MedianExpression } from '@sagebionetworks/agora/api-client-angular';
+
+@Component({
+  selector: 'agora-median-barchart',
+  standalone: true,
```
Comment (tschaffter) [inline libs/agora/charts/src/lib/median-barchart/median-barchart.component.ts:22]: Note: This option is the default in Angular 19 released in November. We will be able to remove them once we upgrade to that version.

**Review by tschaffter — APPROVED** [review]
The stack builds and deploys successfully in the local environment. The migration of the app to the monorepo is progressing smoothly.

**Comment by tschaffter** [comment]
@sagely1 I will go ahead and merge this PR to make sure it's merged before I'm OOO.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
[the remark this answers is not in the document]
Comment (sagely1) [reply .vscode/launch.json:1]: It appeared after Run > Start Debugging (or using shortcut: F5)

File: libs/agora/charts/src/lib/median-barchart/median-barchart.component.ts (hunk 1 above)
Comment (sagely1) [reply libs/agora/charts/src/lib/median-barchart/median-barchart.component.ts:22]: I am looking forward to it!

**Pull request merged**